commands-yml/commands/session/timeouts/timeout.yml

```diff
@@ -0,0 +1,82 @@
+---
+name: Set Timeouts
+short_description: Configure the amount of time that a particular type of operation can execute for before they are aborted
+
+description: 
+  |
+    The types of timeouts are 'page load', 'script' and 'implicit'. (The example usage is just 'page load')
+   
+example_usage:
+  java:
+    |
+       driver.manage().timeouts().pageLoadTimeout(30, TimeUnit.SECONDS);
```

Comment:
Why just page load timeout?

Reply:
The other timeouts are documented in implicit-wait.yml and async-script.yml